var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -30,9 +30,13 @@ class Dealii(Package):
     """C++ software library providing well-documented tools to build finite
     element codes for a broad variety of PDEs."""
     homepage = "https://www.dealii.org"
-    url      = "https://github.com/dealii/dealii/releases/download/v8.4.0/dealii-8.4.0.tar.gz"
+    url      = "https://github.com/dealii/dealii/releases/download/v8.4.1/dealii-8.4.1.tar.gz"
```

Comment:
@davydden true but just wanted to keep that line up to date as well. Would it make a difference that I have changed it?